helm/app-operator-chart/templates/configmap.yaml

```
@@ -18,6 +18,8 @@ data:
       appcatalog:
         index:
           namespace: '{{ .Values.namespace }}'
+      image:
+        registry: '{{ .Values.Installation.V1.Registry.Domain }}' 
```

if it's subject to a domain, why not adding more indent to `domain:`?
```suggestion
        registry: 
          domain:  '{{ .Values.Installation.V1.Registry.Domain }}' 
```

---

I went with image.registry because it matches what we do in helm charts.

The draughtsman values structure is older and I'm not sure we need to follow it exactly. WDYT?

---

No blocker, usually I got the same comment on the helm chart, but if it's older, nevermind.